persistence-cassandra/javadsl/src/main/scala/com/lightbend/lagom/internal/javadsl/persistence/cassandra/CassandraConfigProvider.scala

```diff
@@ -29,15 +29,16 @@ final class CassandraConfigProvider @Inject() (system: ActorSystem) extends Prov
   override lazy val get: CassandraConfig = CassandraConfigProvider.CassandraConfigImpl(cassandraUrisFromConfig)
 
   private def cassandraUrisFromConfig: PSet[CassandraContactPoint] = {
-    val contactPoints = List("cassandra-journal", "cassandra-snapshot-store", "lagom.persistence.read-side.cassandra").flatMap { path =>
-      val c = config.getConfig(path)
-      if (c.getString("session-provider") == classOf[ServiceLocatorSessionProvider].getName) {
-        val name = c.getString("cluster-id")
-        val port = c.getInt("port")
-        val uri = new URI(s"tcp://127.0.0.1:$port/$name")
-        Some(CassandraContactPoint.of(name, uri))
-      } else None
-    }.toSet
+    val contactPoints: Set[CassandraContactPoint] =
+      List("cassandra-journal", "cassandra-snapshot-store", "lagom.persistence.read-side.cassandra").flatMap { path =>
+        val c = config.getConfig(path)
+        if (c.getString("session-provider") == classOf[ServiceLocatorSessionProvider].getName) {
+          val name = c.getString("cluster-id")
+          val port = c.getInt("port")
+          val uri = new URI(s"tcp://127.0.0.1:$port/$name")
+          Some(CassandraContactPoint.of(name, uri))
+        } else None
+      }.toSet
```

Comment:
Tangential to this PR...

This is the first time I've noticed this class, but it seems totally weird! Everything is hardcoded to 127.0.0.1, and this merges all of the contact points together from all of the Cassandra session settings? That would be nonsense if they are actually configured to use different Cassandra clusters or ports.

I poked around, and `CassandraConfig`, while part of the public API, doesn't seem to be actually used internally, or mentioned in the docs. Does anyone know what this is meant to be for? Is it just a mistake or something left over that we should deprecate?